Kickstarter-iOS/Views/Controllers/PaymentMethodsViewControllerTests.swift

```diff
@@ -47,6 +46,8 @@ class PaymentMethodsViewControllerTests: TestCase {
                                                            orientation: .portrait,
                                                            child: controller)
 
+                        parent.view.layoutIfNeeded()
```

Comment:
This was required for the table view header to size correctly in the screenshots. Go figure 😒